STATUS: INLINE FAIL
MODIFIED: pitt.search.semanticvectors.vectors.ComplexVectorTest.testComplexOrthogonalize
MODIFIED FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/vectors/ComplexVectorTest.java
CLEANER: pitt.search.semanticvectors.integrationtests.RegressionTests.testBuildAndSearchComplexPermutationIndex
CLEANER FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/integrationtests/RegressionTests.java
POLLUTER: pitt.search.semanticvectors.CompareTermsTest.testCompareTermsComplexFlatOrthographic
POLLUTER FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/CompareTermsTest.java
ORIGINAL CLEANER SIZE: 6
NEW CLEANER SIZE: 1
ELAPSED TIME: 16.108999967575073
==========================
@@ -224,0 +224,3 @@
+ {
+     int peterRank = positionalBuildSearchGetRank("-dimension 200 -vectortype complex -seedlength 10 -positionalmethod permutation -luceneindexpath positional_index", "-searchtype permutation -queryvectorfile elementalvectors.bin -searchvectorfile permtermvectors.bin simon ?", new String[] { "elementalvectors.bin", "permtermvectors.bin", "docvectors.bin" }, "peter");
+ }
